src/bin/pg_autoctl/fsm.c

```diff
@@ -406,9 +416,8 @@ keeper_fsm_step(Keeper *keeper)
 	 */
 	if (!keeper_update_pg_state(keeper))
 	{
-		log_error("Failed to update the keeper's state from the local "
-				  "PostgreSQL instance, see above for details.");
-		return false;
```

Comment:
Why did you remove this `return false`?

Reply:
The `keeper_fsm_step` function is only used outside of the main loop, from a client side pg_autoctl process. In that context, we don't want to error out when Postgres is not running just because we want to e.g. drop the local node.